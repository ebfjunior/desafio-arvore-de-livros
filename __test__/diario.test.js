import React from 'react';

import Diario from '../src/components/diario';

describe('Componente DiÃ¡rio', () => {
  it('Deve renderizar adequadamente', () => {
    const dia = {"2016-10-12" : 1};
    
    const wrapper = mount(<Diario ano="2016" mes="10" semana="3" periodo={dia}/>);

    expect(wrapper).toMatchSnapshot();
  });

  it('Estado deve ser inicializado corretamente', () => {
    const dia = {"2016-10-12" : 1};
    
    const wrapper = mount(<Diario ano="2016" mes="10" semana="3" periodo={dia}/>);

    expect(wrapper.state().periodo).toEqual(dia);
  });

});