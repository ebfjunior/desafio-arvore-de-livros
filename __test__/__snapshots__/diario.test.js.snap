// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Componente Di√°rio Deve renderizar adequadamente 1`] = `
<Diario
  ano="2016"
  mes="10"
  periodo={
    Object {
      "2016-10-12": 1,
    }
  }
  semana="3"
>
  <svg
    height="20"
    width="20"
    x="0"
    xmlns="http://www.w3.org/2000/svg"
  >
    <rect
      fill="#3FC0D9"
      height="20"
      width="20"
    />
  </svg>
</Diario>
`;
